packages/core/src/AppSwitcherPanel/Action/Action.js

```diff
@@ -0,0 +1,204 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+
+import clsx from "clsx";
+import { useTheme, withStyles } from "@material-ui/core";
+
+import { Info } from "@hitachivantara/uikit-react-icons";
+import { HvTypography, HvAvatar, HvTooltip, HvListItem } from "../..";
+
+import styles from "./styles";
+
+import TitleWithTooltip from "../TitleWithTooltip";
+
+const getColor = (theme, color, defaultColor) => theme.palette[color] || color || defaultColor;
+
+const Action = (props) => {
+  const {
+    id,
+    className,
+    classes,
+    application,
+    onClickCallback = () => {},
+    isSelectedCallback = () => false,
+  } = props;
+
+  const { name, description, disabled, iconElement, iconUrl, url, target } = application;
+
+  const theme = useTheme();
+  const color = disabled
+    ? theme.hv.palette.atmosphere.atmo5
+    : getColor(theme, application.color, theme.hv.palette.accent.acce1);
+
+  const [validIconUrl, setValidIconUrl] = useState(true);
+
+  const renderApplicationIcon = () => {
+    if (iconElement) {
+      return iconElement;
+    }
+
+    if (iconUrl && validIconUrl) {
+      return (
+        // eslint-disable-next-line jsx-a11y/alt-text
```

Comment:
Maybe we can avoid this disable by having the `description=""` as default or in alternative maybe using the role="presentation" as the image may not be critical for accessibility it is just decoration